### PR TITLE
Add Tenant Usage API

### PR DESCRIPTION
**Pull request by dvaldivia**

### CODE REVIEW

File: restapi/admin_tenants.go [hunk 1]
```diff
@@ -111,6 +111,15 @@ func registerTenantHandlers(api *operations.ConsoleAPI) {
 		}
 		return admin_api.NewTenantAddZoneCreated()
 	})
+
+	api.AdminAPIGetTenantUsageHandler = admin_api.GetTenantUsageHandlerFunc(func(params admin_api.GetTenantUsageParams, session *models.Principal) middleware.Responder {
+		err, _ := getTenantUsageResponse(session, params)
+		if err != nil {
+			log.Println(err)
+			return admin_api.NewTenantAddZoneDefault(500).WithPayload(&models.Error{Code: 500, Message: swag.String("Unable to update tenant")})
```
Comment (cesnietor) [inline restapi/admin_tenants.go:119]: wrong error messag also above XD

File: restapi/admin_tenants.go [hunk 2]
```diff
@@ -111,6 +111,15 @@ func registerTenantHandlers(api *operations.ConsoleAPI) {
...
+		}
+		return admin_api.NewTenantAddZoneCreated()
```
Comment (cesnietor) [inline restapi/admin_tenants.go:121]: wrong api response

File: restapi/admin_tenants.go (hunk 1 above)
Comment (cesnietor) [inline restapi/admin_tenants.go:119]: wrong Default function

File: portal-ui/bindata_assetfs.go [hunk 3]
```diff
@@ -314,7 +314,7 @@ func buildAssetManifestJson() (*asset, error) {
 		return nil, err
 	}
 
-	info := bindataFileInfo{name: "build/asset-manifest.json", size: 1324, mode: os.FileMode(420), modTime: time.Unix(1595632751, 0)}
```
Comment (cesnietor) [inline portal-ui/bindata_assetfs.go:317]: this file is no needed in this PR

Comment (dvaldivia) [reply portal-ui/bindata_assetfs.go:317]: it was generated this way by `make assets` which implies it's not in sync

**Pull request merged**